frontend/src/shared/components/tasque-task-editing/task-editing.component.ts

```diff
@@ -315,7 +327,7 @@ export class TaskEditingComponent extends BaseComponent implements OnInit {
       stateId: this.editTaskForm.controls.status.value.id,
       typeId: this.editTaskForm.controls.type.value.id,
       projectId: this.editTaskForm.controls.project.value.id,
-      //sprintId: this.editTaskForm.controls.sprint.value.id,
+      sprintId: this.editTaskForm.controls.sprint.value.id !== 0 ? this.editTaskForm.controls.sprint.value.id : undefined,
```

Comment:
There is not much sence is using comparison with `zero` here - in case id equals `zero`, it will return `false` as boolean

Reply:
That's what i needed)